h/static/scripts/annotation-mapper.js

```diff
@@ -45,6 +70,10 @@ function annotationMapper($rootScope, threading, store) {
 
       $rootScope.$emit('annotationDeleted', annotation);
     });
+
+    annotations.slice().forEach(function(annotation) {
+      loaded.splice(loaded.indexOf(annotation, 1));
+    });
```

Comment:
This crazy dance is because when our `$rootScope.$on('groupFocused', ...` above calls `unloadAnnotations()`, the passed `annotations` array _is_ the `loaded` array, and you can't modify an array while you iterate over it in JavaScript, so we need to iterate over a copy of it.